web/controllers/post_controller.ex

```diff
@@ -25,7 +25,7 @@ defmodule Healthlocker.PostController do
   end
 
   def index(conn, _params) do
-    posts = Repo.all(Post)
+    posts = Repo.all(from p in Post, where: ilike(p.content, "%#story%"))
```

Comment:
For instance the controller test we have (which is good) doesn't really test whether this query works or not? By moving it into the Model is keeps the responsibility for querying in the model, and we can independently test it there e.g. create a number of posts some with the correct tag, and some without, and then tests that the results returned are what we expect.